Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/IMixedRealityInputSource.cs

```diff
@@ -12,17 +12,41 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
     /// </summary>
     public interface IMixedRealityInputSource : IEqualityComparer
     {
+        /// <summary>
+        /// The ID assigned to the Controller
+        /// </summary>
         uint SourceId { get; }
 
+        /// <summary>
+        /// The Name assigned to the Controller
+        /// </summary>
         string SourceName { get; }
 
+        /// <summary>
+        /// Outputs the current state of the controller, whether it is tracked or not. As defined by the SDK / Unity.
+        /// </summary>
+        ControllerState ControllerState { get; }
```

Comment:
Let's rename to SourceState or similar.

Reply:
Agreed

Reply:
Renamed to InputSourceState, including enum